<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QWidget" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/MainWindow/Resources/replace.svg</normaloff>:/MainWindow/Resources/replace.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonReplace">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Replace to:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editReplaceWhat"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Replace what:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editReplaceTo"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkMatchCase">
       <property name="text">
        <string>Match case</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editReplaceWhat</tabstop>
  <tabstop>editReplaceTo</tabstop>
  <tabstop>buttonReplace</tabstop>
  <tabstop>checkMatchCase</tabstop>
 </tabstops>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
